scanner/datastore/postgres/external_index_report.go

```diff
@@ -0,0 +1,91 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/quay/claircore"
+	"github.com/quay/zlog"
+)
+
+func (e *externalIndexStore) StoreIndexReport(ctx context.Context, hashID string, clusterName string, indexReport *claircore.IndexReport) error {
+	ctx = zlog.ContextWithValues(ctx, "component", "datastore/postgres/externalIndexStore.StoreIndexReport")
+
+	const insertIndexReport = `
+		INSERT INTO external_index_report (hash_id, clusterName, indexReport) VALUES
+			($1, $2, $3)
+		ON CONFLICT (hash_id, clusterName) DO UPDATE SET indexReport = $3`
```

Comment:
On an update should we reset the existing expiration so that it doesn't get deleted shortly after?

Reply:
The expectation is that this function (`StoreIndexReport()`) will be used for instances where reindexing is disabled. Otherwise, `StoreIndexReportWithExpiration()`, which should set that expiration like you suggest, will be used.